web/css/chat-room.css

```diff
@@ -0,0 +1,311 @@
+
```

Comment:
Sorry I'm picky with indentation, but better readability helps find issues, so it's part of writing good code.

Our linter doesn't check CSS, but we want indentation to be correct in CSS too.